### PR TITLE
export and import constrains

### LINKED ISSUE #65: Handle constraints during export/import of content

### PR DESCRIPTION
**Pull request by pbauer**
Fixes #65

### CODE REVIEW

File: src/collective/exportimport/export_content.py [hunk 1]
```diff
@@ -421,6 +424,18 @@ def fix_url(self, item, obj):
...
 
+    def export_constraints(self, item, obj):
+        constrains = ISelectableConstrainTypes(obj, None)
+        if constrains is None:
+            return item
+        if constrains.getConstrainTypesMode() == ENABLED:
+            key = "exportimport.constrains"
+            item[key] = {
+                "locally_allowed_types": constrains.getLocallyAllowedTypes(),
+                "immediately_allowed_types": constrains.getImmediatelyAddableTypes(),
```
Comment (mauritsvanrees) [inline src/collective/exportimport/export_content.py:435]: For clarity, could you rename this to `immediately_addable_types`?

**Review by mauritsvanrees — APPROVED** [review]
Tested with export and import on Plone 4.3, and it works. Thanks! At first I thought we would miss the constrainTypesMode field itself, but this is automatically contained in the schema.

One minor inline comment, but is also fine to merge as is.

**Comment by pbauer** [comment]
Thanks for the review. I now already had a case where the constrains didi not allow importing the exported data, e.g.  because a folder or a collection was added and then later disallowed. 
How about adding content disregarding the constraints e.g. using `plone.dexterity.utils.createContentInContainer` or `Products.CMFCore.TypesTool._constructInstance`?

**Comment by mauritsvanrees** [comment]
Ah. Yes, that seems good.

**Pull request merged**